packages/components/src/card/context.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * WordPress dependencies
+ */
+import { createContext, useContext } from '@wordpress/element';
+
+export const CardContext = createContext( {} );
+export const useCardContext = () => useContext( CardContext );
+
+export const CardProvider = ( props ) => {
```

Comment:
What value does this component add over just using the `CardContext.Provider` one?

Reply:
Good question!

This was a pattern I started using as the `Provider` became more complex. It was an abstraction to keep the initial entry component (the `index.js`) code relatively simple. Sorta like how you might have a simple `app.js`, but a separated complicated Redux action/reducer setup.

Since the current implementation + feature-set is very minimal, I can consolidate the Provider logic into the main `index.js` file :). No need to pre-optimize - also, the code is clearer 👍